apps/catalogue/src/main.js

```diff
@@ -397,21 +399,61 @@ const router = new VueRouter({
     },
     {
       name: "NetworkLandingPage",
-      path: "/networks-new",
+      path: "/networks-catalogue",
       component: NetworksHome,
     },
+    {
+      name: "NetworkDetails",
+      path: "/networks-catalogue/:network",
+      props: true,
+      component: NetworkDetails,
+    },
     {
       name: "NetworkVariables",
-      path: "/networks-new/:network",
+      path: "/networks-catalogue/:network/variables",
       props: true,
       component: NetworkVariables,
     },
     {
       name: "NetworkVariableDetailView",
-      path: "/networks-new/:network/:name",
+      path: "/networks-catalogue/:network/variables/:name",
       props: (route) => ({ ...route.params, ...route.query }), // both key and value are dynamic
       component: VariableDetailView,
     },
+    {
+      name: "NetworkCohorts",
+      path: "/networks-catalogue/:network/cohorts",
+      props: true,
+      component: NetworkCohorts,
+    },
+    {
+      name: "NetworkCohortDetailView",
+      path: "/networks-catalogue/:network/cohorts/:pid",
+      props: true,
+      component: CohortView,
+    },
+    {
+      name: "NetworkCohortSubcohort",
+      path: "/networks-catalogue/:network/cohorts/:cohort/subcohorts/:name",
+      props: true,
+      component: Subcohort,
+    },
+    {
+      name: "NetworkCohortCollectionEvent",
+      path: "/networks-catalogue/:network/cohorts/:cohort/collection-events/:name",
+      props: true,
+      component: CollectionEvent,
+    },
+    {
+      // hacky redirects to solve breadcrumb issue
+      path: "/networks-catalogue/:network/cohorts/:cohort/collection-events",
+      redirect: "/networks-catalogue/:network/cohorts/:cohort",
+    },
+    {
+      // hacky redirects to solve breadcrumb issue
+      path: "/networks-catalogue/:network/cohorts/:cohort/subcohorts",
+      redirect: "/networks-catalogue/:network/cohorts/:cohort",
```

Comment:
why would you need these paths ?

Reply:
When viewing subcohorts of a cohort in context of a network. So to keep network id in the path.

Reply:
suggest to add "When viewing subcohorts of a cohort in context of a network. So to keep network id in the path." as it is a more descriptive comment